src/removeTask.js

```diff
@@ -0,0 +1,6 @@
+const removeTask = (id, tasks) => {
+  const newTasks = tasks.filter((task) => task.id !== id);
+  return newTasks;
+};
+
+export default removeTask;
```

Comment:
- [ ] Something similar is happening with the approach you chose for deleting tasks, as shown in the video, when the three dots are clicked not only does the content of the task becomes an input, but a thrash icon appears in place of the dots and once this icon is clicked, the task is deleted. Currently, you implemented the `Clear Completed` button, however, this will be done in the next milestone. Please consider following the given approach! 🤓 🔧 

- [ ] I also noticed that when a task is deleted, indexes don't update, according to the requirements:

![image](https://user-images.githubusercontent.com/77069011/236359247-95cf7323-d016-42bc-a673-c668d9fe653f.png)

Please consider implementing this update! 🤓 🔧